### PR TITLE
Fix issue 319

### PR DESCRIPTION
**Pull request by mtoma**

### CODE REVIEW

File: lib/Zonemaster/Backend/TestAgent.pm [hunk 1]
```diff
@@ -58,7 +58,7 @@ sub run {
     }
     $domain = $self->to_idn( $domain );
 
-    if (defined $params->{ipv4} || defined $params->{ipv4}) {
```
Comment (mattias-p) [inline lib/Zonemaster/Backend/TestAgent.pm]: Could you update this file to be consistent about spaces and tabs in the indentation?

**Comment by sandoche2k** [comment]
@mtoma please merge after resolving the conflicts

**Comment by mattias-p** [comment]
When resolving the conflict, please also exclude my merge commit from this PR. It doesn't belong here.

**Comment by sandoche2k** [comment]
@matsduf @mattias-p I see only two commits with respect to #319 and indentation.

**Pull request merged**